src/Apps/Sell/sellRoutes.tsx

```diff
@@ -284,6 +267,24 @@ export const sellRoutes: RouteProps[] = [
               return { id }
             },
           },
+          {
+            path: "thank-you",
+            layout: "ContainerOnly",
+            Component: ThankYouRoute,
+            onClientSideRender: () => {
+              ThankYouRoute.preload()
+            },
+            query: graphql`
+              query sellRoutes_ThankYouRouteQuery($id: ID!) {
+                submission(id: $id) @principalField {
+                  ...ThankYouRoute_submission
+                }
+              }
+            `,
+            prepareVariables: ({ id }) => {
+              return { id }
+            },
+          },
```

Comment:
Moving the thank-you route inside of the steps to render the progress bar for this step as well.